### ISSUE TITLE
Cutting Board and other Items changing texture when placed

### ISSUE THREAD

**Issue by AranciaCibo** [vectorwing/FarmersDelight#1012]
### Minecraft version

1.20.1

### Farmer's Delight version

1.2.5

### NeoForge version

47.3.12

### Description

When I place down a farmer's delight mod item, for example, the cutting board, its texture transforms into the Onion Crate, the same goes with other items, the cabinets change, for example, when I place an oak cabinet, it becomes a spruce cabinet, and so on, also, when broken, it once again becomes the original item when dropped. Also, this issue is only available on a multiplayer server with my friend, when I tested it on singleplayer, it worked normally.

### Steps to reproduce

Just placing the item

### Mod list

WTHIT
Advanced Netherite
Apple Skin
Better Advancements
Better Villages
Guard Villagers
Journeymap
Gravestone
Serene Seasons
Traveler's Backpack
Waystones

### Minimal instance

- [X] I have tested this on a minimal instance

### Performance and shader mods

- [ ] I am using performance or shader mods

**Comment by vectorwing** [comment]
Since you confirmed you are not using performance or shader mods, this could be caused by a version mismatch between your client and the server.

Make sure both are using the same versions of your mods, and try again.

**Comment by gemerat** [comment]
Same for me, the versions are the same in both sides. Its only fail in multiplayer server. I use 1.20.1 too

**Comment by Caleb5904** [comment]
i also have the same problem. i've tried with and without shaders too
![2024-12-01_23 02 35](https://github.com/user-attachments/assets/4f13facd-b020-4747-aad7-6bae283b3164)
![2024-12-01_23 02 51](https://github.com/user-attachments/assets/8977353e-a55b-491a-93ef-16d444ba4c1f)

**Comment by Caleb5904** [comment]
[gemerat] I figured it out. he's not talking about the game version (1.20.1) he's talking about the mod itself
so for me on forge it'd be (farmers delight 1.20.1 1.2.5) and if the server has (farmers delight 1.20.1 1.2.4) it'll cause the "bug"

**Comment by vectorwing** [comment]
> [gemerat] I figured it out. he's not talking about the game version (1.20.1) he's talking about the mod itself so for me on forge it'd be (farmers delight 1.20.1 1.2.5) and if the server has (farmers delight 1.20.1 1.2.4) it'll cause the "bug"

Is the issue fixed if you match the mod's versions between client and server?
This kind of mismatch usually happens due to registry mapping differences.

**Comment by Caleb5904** [comment]
> > [gemerat] I figured it out. he's not talking about the game version (1.20.1) he's talking about the mod itself so for me on forge it'd be (farmers delight 1.20.1 1.2.5) and if the server has (farmers delight 1.20.1 1.2.4) it'll cause the "bug"
> 
> 
> 
> Is the issue fixed if you match the mod's versions between client and server?
> 
> This kind of mismatch usually happens due to registry mapping differences.

Yes! So my problem was; I had Farmers Delight Version 1.2.5 installed on forge. I didn't pay attention enough and downloaded Farmers Delight Version 1.2.4 on my server. Took me a bit to understand what you said about matching the "Versions" but I figured it out and installed Farmers Delight Version 1.2.5 on my server and deleted 1.2.4 and it work just how it's supposed to!

**Comment by vectorwing** [comment]
@AranciaCibo and @gemerat, could you two please verify with the server owners if the versions of FD match? I am confident this is the solution, but I need to confirm from your ends first.

To clarify: I mean the *mod* version, not Minecraft's. 1.2.4, 1.2.5 etc.

**Comment by gemerat** [comment]
It's working now, I haven't changed anything, I just removed some mods like handcrafted and downloaded the Patchouli mod. I think the patchouli has been key. I'm new in this world and there are things that can escape me. Thanks for your replies,

**Comment by vectorwing** [comment]
Glad to know it's resolved! I'll be closing the issue for now, since AranciaCibo hasn't replied in a month. Let me know if the issue persists.
